plugins/scaffolder/src/next/OngoingTask/OngoingTask.tsx

```diff
@@ -132,15 +133,12 @@ export const OngoingTask = (props: {
         ) : null}
 
         <Box paddingBottom={2}>
-          <Paper style={{ position: 'relative', overflow: 'hidden' }}>
-            <TaskBorder
```

Comment:
Thought that this component was pretty useless on it's own, so wrapped it all up in the paper instead and just export that @dagda1

Reply:
Or do you think it would be useful to be able to access just the steps on their own?

Reply:
@benjdlambert I think this makes sense.  Having the two components separate is a nothing gain and maybe more confusing